testsuite/do_tests.sh.in

```diff
@@ -724,13 +724,13 @@ if test "x${TEST_PYNEST}" = xtrue ; then
     #    of support for nosetests. We use the TEST_OUTDIR as Python-free
     #    dummy directory to search for tests.
 
-    if command -v nosetests >/dev/null 2>&1 && nosetests --with-xunit --xunit-file=/dev/null --where=${TEST_OUTDIR} >/dev/null 2>&1; then
+    if command -v @NOSETESTS@ >/dev/null 2>&1 && @NOSETESTS@ --with-xunit --xunit-file=/dev/null --where=${TEST_OUTDIR} >/dev/null 2>&1; then
```

Comment:
This line checks
1. if the `nosetests` executable is available (`command -v` yields the same as which, but is more portable) and
2. if it can be run successfully using the options we are using below in line 733
I think that 2. should also use the correct Python to be meaningful